db/migrate/20211018093925_create_profiles.rb

```diff
@@ -0,0 +1,8 @@
+class CreateProfiles < ActiveRecord::Migration[6.1]
+  def change
+    create_table :profiles do |t|
```

Comment:
do you need this table at all if it is empty?

Reply:
> 

i will add some fields in the future

Reply:
Follow the YAGNI principle

https://betterprogramming.pub/yagni-you-aint-gonna-need-it-f9a178cd8e1